### ISSUE TITLE
can not observe textarea text change but work well with input

### ISSUE THREAD

**Issue by CocaCola183** [artf/cimice#1]
my own keyup event calback:

```js
keyupEventCallback(recorder, movie, e) {
    let focusedElement = document.activeElement;
    focusedElement.setAttribute('value', focusedElement.value);

    const t = e.target;
    const b = t.body;
    const scrollTop = b ? (b.parentNode.scrollTop || b.scrollTop) : t.scrollTop;
    const scrollLeft = b ? (b.parentNode.scrollLeft || b.scrollLeft) : t.scrollLeft;
    movie.addFrame({
      scrollY: scrollTop,
      scrollX: scrollLeft,
      cursorX: e.pageX,
      cursorY: e.pageY,
      width: recorder.target.clientWidth,
      height: recorder.target.clientHeight,
      eventType: e.type,
    });
  }
```

works fun with input, but not work with textarea.

something wrong?

**Comment by artf** [comment]
I'm wondering how does it work with inputs if you aren't adding additional stuff to the frame (in addFrame)?!?

Anyway, I've tried with the `keyup` event and everything works as expected
From your example I'd expect something like this:
```js
keyupEventCallback(recorder, movie, e) {
    let focusedElement = document.activeElement;
    
    // The value is already setted, you don't need this
    //focusedElement.setAttribute('value', focusedElement.value);
 
   // This, at any keyup, will show the actual value
   var actualInputValue = focusedElement.value;
   console.log(actualInputValue);

    const t = e.target;
    const b = t.body;
    const scrollTop = b ? (b.parentNode.scrollTop || b.scrollTop) : t.scrollTop;
    const scrollLeft = b ? (b.parentNode.scrollLeft || b.scrollLeft) : t.scrollLeft;
    movie.addFrame({
      scrollY: scrollTop,
      scrollX: scrollLeft,
      cursorX: e.pageX,
      cursorY: e.pageY,
      width: recorder.target.clientWidth,
      height: recorder.target.clientHeight,
      eventType: e.type,
      inputValue: actualInputValue, // <- make the frame know about the value
    });
  }
```

Of course, after the recorder you've also to setup the player:
```js
player.on('keyup', function(frame){
 console.log(frame.inputValue);
 // ...
});
```

**Comment by CocaCola183** [comment]
@artf 

Sorry for some misunderstanding of your project and appreciate for your time, but I still have questions. 

**to answer your wander**
recorder target is default.
When I typing something in <input />, I got keyup event emitted, but if I do not change the `abstractEventCallback` like above, recorder would add the exactly same frame just like noting happened, so I would never see any changes(what I'm typing) in the movie.

I just fixed this issue in this way:
```js
let focusedElement = document.activeElement;
if (focusedElement.tagName == 'TEXTAREA') {
  focusedElement.innerHTML = focusedElement.value;
} else {
  focusedElement.setAttribute('value', focusedElement.value);
}
```

**doubt 1**
And about your answer above
```js
player.on('keyup', function(frame){
 console.log(frame.inputValue);
 // ...
});
```
Even though I got the `inputValue`, how could I know which input source is on focus?

**doubt 2**
Any suggestions for recording subnodes'(like a div) scroll?

Appreciate for your time again.
